tools/clang/tools/dxcompiler/dxcshadersourceinfo.h

```diff
@@ -56,7 +56,7 @@ struct SourceInfoWriter {
   Buffer m_Buffer;
 
   const hlsl::DxilSourceInfo *GetPart() const;
-  void Write(llvm::StringRef targetProfile, llvm::StringRef entryPoint, clang::CodeGenOptions &cgOpts, clang::SourceManager &srcMgr);
+  void Write(clang::CodeGenOptions &cgOpts, llvm::ArrayRef<hlsl::options::ArgPair> ArgPairs, clang::SourceManager &srcMgr);
```

Comment:
Any reason this uses the full `hlsl::optons::ArgPair` while other instances in this file use just `options::ArgPair`?

Reply:
Removed it. That might have been from copypasting from other places.